### ISSUE TITLE
Duplicates seem broken

### ISSUE THREAD

**Issue by miketaylr** [mozilla/tsci#83]
At https://docs.google.com/spreadsheets/d/1rmH1EhTVZBtTIXXwqSHcaQXhmv7xJh2Y2LuNiySPURQ/edit?ts=5cf460c1#gid=1551760774, there are no duplicates (except for one for espn.com). That seems broken.

**Comment by miketaylr** [comment]
I think I know what did this -- trying to fix in #82

**Comment by miketaylr** [comment]
> I think I know what did this -- trying to fix in #82

No, I thought I broke the Bugzilla query -- but it's working as expected. What seems to be broken is that GitHub no longer returns issues from the search API if you search for issue numbers?

https://api.github.com/search/issues?q=is%3Aissue+milestone%3Aduplicate+repo%3Awebcompat%2Fweb-bugs%2F+14696+10398+14809+14964+14999+14981+14984+15154+15930+16212+16517+16668+16760+17265+17385+17719+18804+19462+19558+19682+20277+21244+23675+23359+23764+23967+23980+24508+24626+24653+26737+27376 returns 0 issues now. :(

**Comment by miketaylr** [comment]
If you log out `query` and `ghToBzMap`, you see expected results:

```
[
    'is%3Aissue+milestone%3Aduplicate+repo%3Awebcompat%2Fweb-bugs%2F+14696+10398+14809+14964+14999+14981+14984+15154+15930+16212+16517+16668+16760+17265+17385+17719+18804+19462+19558+19682+20277+21244+23675+23359+23764+23967+23980+24508+24626+24653+26737+27376',
    Map {
      14696 => 975444,
      10398 => 975444,
      14809 => 975444,
      14964 => 975444,
      14999 => 975444,
      14981 => 975444,
      14984 => 975444,
      15154 => 975444,
      15930 => 975444,
      16212 => 975444,
      16517 => 975444,
      16668 => 975444,
      16760 => 975444,
      17265 => 975444,
      17385 => 975444,
      17719 => 975444,
      18804 => 975444,
      19462 => 975444,
      19558 => 975444,
      19682 => 975444,
      20277 => 975444,
      21244 => 975444,
      23675 => 975444,
      23359 => 975444,
      23764 => 975444,
      23967 => 975444,
      23980 => 975444,
      24508 => 975444,
      24626 => 975444,
      24653 => 975444,
      26737 => 975444,
      27376 => 975444
    }
  ],
```
